app/views/landing_page/landing_page.erb

```diff
@@ -57,6 +57,10 @@
 
   <title><%= page["title"]["value"] %></title>
 
+  <% unless local_assigns.has_key?(:skip_meta) && skip_meta %>
```

Comment:
Is there any reason to have the option to skip meta? This part depends heavily on the structure json so it would help to have the editor preview render it too to catch problems in that part of the json. Providing fake data to resolver shouldn't be a problem in editor, we do that already, just need to extend the fake provider.